src/cli/tact-fmt/index.ts

```diff
@@ -80,7 +82,11 @@ type Args = ArgConsumer<GetParserResult<ReturnType<typeof ArgSchema>>>;
 
 const markup = getAnsiMarkup(Boolean(isColorSupported()));
 
-function formatFile(filepath: string, write: boolean): boolean | undefined {
+function formatFile(
+    filepath: string,
+    write: boolean,
+    onlyCheck: boolean,
```

Comment:
Looks like this should be one `mode` parameter instead of these two. For instance, `write` and `onlyCheck` both `true` do not make sense.

Reply:
fixed